dwellingly/src/App.js

```diff
@@ -87,6 +88,7 @@ export class App extends React.Component {
               <PrivateRoute exact path='/' component={Dashboard} />
               <Route exact path='/login' component={LoginForm} />
               <PrivateRoute exact path='/dashboard' component={Dashboard} />
+              <Route exact path='/add-property' component={AddProperty} />
```

Comment:
Since we should have persistent auth merged in soon, we should make this a "PrivateRoute" since you need to be authenticated to use it